### PR TITLE
Landing Page and Card Components

### PR DESCRIPTION
**Pull request by dhruvin25799**
## Landing Page and Card Components.

Added Landing Page with styles and card components.
Navbar and Footer were also added.
Live : https://bucolic-jelly-04f990.netlify.app/

## Files to check : 

- [ ] src/components/Button/Button.jsx
- [ ] src/components/Footer/Footer.jsx
- [ ] src/components/NavBar/NavBar.jsx
- [ ] src/hooks/useToggle.js
- [ ] src/pages/Home/Home.jsx

## Files to ignore : 

- [ ] All Card components as they were reviewed previously.
- [ ] All CSS files since they're the same as screens and reviewed previously.

### CODE REVIEW

File: src/hooks/useToggle.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useState } from "react"
```
Comment (PuneetP16) [inline src/hooks/useToggle.js:1]: You could keep separate folder for custom hooks.

File: src/pages/Home/Home.module.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  color: var(--COLOR-secondary-text);
+  font-size: 1rem;
+  width: 50%;
+}
+
+
+
+
+.main {
+  display: flex;
```
Comment (PuneetP16) [inline src/pages/Home/Home.module.css:36]: remove extra empty lines, use some code formatter as a last resort

File: src/pages/Home/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.main-cta-content h1 {
+  color: var(--COLOR-primary-green);
+  font-size: 2.5rem;
+  width: 40%;
+  line-height: 3.5rem;
+}
+.resp-image {
+  width: 100%;
```
Comment (PuneetP16) [inline src/pages/Home/Home.module.css:84]: here too.

**Review by PuneetP16 — APPROVED** [review]
Your UI is too good. Looks good to merge.  Though check the comments

File: src/components/Button/Button.module.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  box-shadow: 0px 2px 3px #595254;
+  padding: 10px 20px 10px 20px;
+  text-decoration: none;
+  border: none;
+  border-radius: 0.5rem;
+  transition: all 0.2s ease 0s;
+  cursor: pointer;
+  background-color: var(--COLOR-primary-green);
+  width: fit-content;
+  color: white;
```
Comment (Ananyamadhu08) [inline src/components/Button/Button.module.css:11]: use variables for color to maintain consistency.

File: src/components/Card/CartCard.jsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faXmark, faPlus, faMinus } from "@fortawesome/free-solid-svg-icons";
+import { RupeeSign } from "../RupeeSign/RupeeSign";
+import styles from "./CartCard.module.css";
+import { Button } from "../Button/Button";
+export const CartCard = (props) => {
```
Comment (Ananyamadhu08) [inline src/components/Card/CartCard.jsx:7]: you can destructure the props to maintain readability

File: src/components/Card/CartCard.module.css [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    flex-flow: row;
+    gap: 1rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    border-radius: 10px;
+    overflow-x: hidden;
+    padding: 1rem;
+}
+
+.cart-card .card-image{
+    width: 200px;
```
Comment (Ananyamadhu08) [inline src/components/Card/CartCard.module.css:12]: use consistent spacing units, if you're using rem use rem throughout

File: src/components/Card/HomeCard.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./HomeCard.module.css"
+import {Button} from "../Button/Button";
+import {Link} from "react-router-dom";
+export const HomeCard = (props) => {
```
Comment (Ananyamadhu08) [inline src/components/Card/HomeCard.jsx:5]: destructuring for readability

File: src/components/Card/HomeCard.module.css [hunk 8]
```diff
@@ -0,0 +1,46 @@
+.card {
+  display: flex;
+  flex: 1 1 50%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  background-color: #e2ede9;
```
Comment (Ananyamadhu08) [inline src/components/Card/HomeCard.module.css:5]: use variables wherever possible

File: src/components/Footer/Footer.module.css [hunk 9]
```diff
@@ -0,0 +1,62 @@
+.footer {
+  padding: 2rem;
+  background-color: black;
```
Comment (Ananyamadhu08) [inline src/components/Footer/Footer.module.css:3]: use variables for colors to maintain consistency.

File: src/components/Footer/Footer.module.css [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+  display: flex;
+  flex-flow: row;
+  justify-content: space-between;
+  color: white;
```
Comment (Ananyamadhu08) [inline src/components/Footer/Footer.module.css:7]: here too

**Review by Ananyamadhu08 — APPROVED** [review]
use consistent variable for colors and consistent spacing other than that your code looks good. Amazing UI! you can go ahead and merge.

**Pull request merged**